config/routes.rb

```diff
@@ -2,266 +2,268 @@
 
 # require "rswag-api"
```

Comment:
This is unrelated with your PR but I think it would be good to delete it as is commended thus, not needed